Lib/test/datetimetester.py

```diff
@@ -399,6 +400,11 @@ def test_harmless_mixed_comparison(self):
         self.assertIn(me, [1, 20, [], me])
         self.assertIn([], [me, 1, 20, []])
 
+        # Comparison over a type other that is not object's type should return
+        # NotImplemented and fallback to other.__eq__. In this case ANY.__eq__
+        # always returns True.
```

Comment:
I find this comment a bit hard to parse (specifically the "Comparison over a type other"...), maybe this instead?

```python
        # Comparison to objects of unsupported types should return
        # NotImplemented which falls back to the right hand side's __eq__
        # method. In this case, ANY.__eq__ always returns True.
```